CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 
 ## Unreleased
 
+- **Kafka Scaler:** Support of passphrase encrypted PKCS #\8 private key [3413](https://github.com/kedacore/keda/pull/3413)
```

Comment:
Please move this to `Improvements Section below`, sort alphabetically and please refer an issue (not PR in the Changelog). If there's not an issue covering this feature request, please create one.